### PR TITLE
PX4 SD Card layout info

### PR DESCRIPTION
**Pull request by hamishwillee**
@bkueng As discussed in slack.

### CODE REVIEW

File: en/concept/sd_card_layout.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+The PX4 SD Card is used for storing configuration files, flight logs, mission information etc. 
+
+> **Note** The file system is FAT32.
+
+The directory structure/layout is shown below.
+
+Directory/File(s) | Description
+--- | ---
+/etc/ | Extra config (+ mixers)
```
Comment (bkueng) [inline en/concept/sd_card_layout.md]: Link to https://dev.px4.io/master/en/concept/system_startup.html#replacing-the-system-startup?

File: en/concept/sd_card_layout.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+/etc/mixers/ | [Mixers](../concept/mixing.md)
+/log/ | Full [flight logs](../log/logging.md)
+/mission_log | Reduced flight logs
```
Comment (bkueng) [inline en/concept/sd_card_layout.md]:
```suggestion
/mission_log/ | Reduced flight logs
```

File: en/concept/sd_card_layout.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# PX4 SD Card Layout
+
+The PX4 SD Card is used for storing configuration files, flight logs, mission information etc. 
+
+> **Note** The file system is FAT32.
```
Comment (bkueng) [inline en/concept/sd_card_layout.md]: Maybe note that this is the default, and you don't have to format before using it?

**Review by bkueng — APPROVED** [review]
Thanks @hamishwillee

File: en/concept/sd_card_layout.md (hunk 3 above)
Comment (hamishwillee) [reply en/concept/sd_card_layout.md]: Yes. That implies you can use other formats if you like. I've updated, but if we can change formats, we should probably link to how/where?

File: en/concept/sd_card_layout.md (hunk 1 above)
Comment (hamishwillee) [reply en/concept/sd_card_layout.md]: Yes!

**Comment by hamishwillee** [comment]
Thanks for the review @bkueng. There is a comment above, but merging this as is OK for now.

File: en/concept/sd_card_layout.md (hunk 3 above)
Comment (bkueng) [reply en/concept/sd_card_layout.md]: Sorry, what do you mean by that?

Comment (hamishwillee) [reply en/concept/sd_card_layout.md]: > Sorry, what do you mean by that?

So you said "Maybe note that this is the default". If something has a default of FAT32, that implies that you might have another file system. Right?

If so, what are the other options for a file system, and how do we set PX4 to use them?

Comment (bkueng) [reply en/concept/sd_card_layout.md]: I see. You can format an SD card to any file system you want. But that's your conscious choice then and you have to make sure you switch back to FAT32 again if you want to use it with PX4 (we don't really support others).

Comment (hamishwillee) [reply en/concept/sd_card_layout.md]: Thanks. So I had assumed that when you said "formatting is not required before use" the card will be formatted with FAT32 automatically by PX4 if it wasn't already. Is that the case.

If not, we should just say use a formatted FAT32 SD card, and ignore any nuances.

Comment (bkueng) [reply en/concept/sd_card_layout.md]: There's logic to reformat the SD card if mouting fails, but I rather suggest users to format themselves (I'm not sure how reliable NuttX does it).

Comment (hamishwillee) [reply en/concept/sd_card_layout.md]: I agree. Done in https://github.com/PX4/Devguide/pull/980.

Thanks.

**Pull request merged**